packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -1112,22 +1116,30 @@ export class GarbageCollector implements IGarbageCollector {
 		outboundRoutes.push(toNodePath);
 		this.newReferencesSinceLastRun.set(fromNodePath, outboundRoutes);
 
-		this.telemetryTracker.nodeUsed({
+		// GC won't recognize some subDataStore paths that we encounter (e.g. a path suited for a custom request handler)
+		// So for subDataStore paths we need to check the parent dataStore for current tombstone/inactive status.
+		const trackedId =
```

Comment:
This is not really needed because anything other than a DDS cannot store handles aka references to other nodes. It's fine to have it this way but just wanted to note that.

Reply:
Not sure I follow. I'm making sure the "to" node that is referenced is handled even if it's an unrecognized path

Reply:
Oh I see,  my bad. I thought this was the fromPath.